dev/Projection/Microsoft.UI.Xaml.Projection.csproj

```
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+      <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+      <ImplicitUsings>enable</ImplicitUsings>
+      <Nullable>enable</Nullable>
+      <Platform>AnyCPU</Platform>
+      <UseUwp>true</UseUwp>
+      <IsAotCompatible>true</IsAotCompatible>
+      <WindowsSdkPackageVersion>10.0.22621.37-preview</WindowsSdkPackageVersion>
+   </PropertyGroup>
+
+   <PropertyGroup>
+      <CsWinRTIncludes>Microsoft.UI.Xaml</CsWinRTIncludes>
+   </PropertyGroup>
+
+   <ItemGroup>
+      <PackageReference Include="Microsoft.Windows.CSWinRT" Version="2.1.1" />
+      <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2739.15">
+         <IncludeAssets>none</IncludeAssets>
+      </PackageReference>
```

Nit: can make this a one liner
```suggestion
      <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2739.15" IncludeAssets="none" />
```